docs/index.rst

```diff
@@ -25,19 +25,43 @@ The vision is that it grows over time as the teams and governance structure evol
 
 .. toctree::
    :maxdepth: 2
-   :caption: Policies
+   :caption: Community Leadership
    :hidden:
 
-   policies/code_of_conduct/code_of_conduct
- 
+   community_leadership/community_leadership
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Governance
+   :hidden:
+
+
 
 .. toctree::
    :maxdepth: 2
    :caption: Policies
    :hidden:
 
+   policies/code_of_conduct/code_of_conduct
```

Comment:
Could you also correct the title of this page so we're using Code of Conduct - I think for a heading it's fine to use both capitalised and we should be consistent with both menu headings otherwise it looks a little odd.